test/js/pageobjects/Welcome.js

```diff
@@ -7,7 +7,7 @@ class WelcomeScreen extends Base{
     get footer() { return this.$('.onfido-sdk-ui-Theme-footer')}
     get primaryBtn() { return this.$('.onfido-sdk-ui-Button-button')}
 
-    copy = (lang="en") =>
+    copyWelcome = (lang="en") =>
```

Comment:
We could import the local only once in the happy.js file so we don't need to repeat this for each page object